zetta_utils/mazepa_layer_processing/common/volumetric_apply_flow.py

```diff
@@ -50,7 +51,68 @@ def __call__(
 
 @mazepa.taskable_operation_cls
 @attrs.mutable
```

Comment:
Should be deleted.

Reply:
Sorry, what should be deleted?  (These decorations are needed, or mypy freaks out.)

Reply:
Oh nvm — the first one is needed, but @attrs.mutable is not, that must be what you meant.